test/unit/request.test.js

```diff
@@ -46,6 +46,15 @@ describe('Request', function () {
             expect(req.toJSON()).to.have.keys(['certificate', 'proxy', 'url']);
         });
 
+        it('should handle falsy request methods correctly', function () {
+            var req = new Request({
+                method: null,
+                url: 'https://postman-echo.com/:path'
+            });
+
+            expect(req.method).to.be('GET');
```

Comment:
Could you also add a test for `new Request().method === 'GET'` ? Since there is additional handling for falsy `options` in constructor?